### DISCUSSION TITLE
Mod Manager not available

### DISCUSSION THREAD

**Discussion by Tetr1sForev3r** [thatskymod/Sky-CotL-Scripts#73] [General]
After I download the canva, the mod manager isn't here. I can only see the canva and when I open it, it gives me the mod manager's options so I can't play at all. 
(Edit : Oh hey nvm!! They just built the mod manager and canva onto one.. We just gotta click the live button, im stupid 😭)

**Comment by YumiAkiyami** [comment]
Same issues here!!

**Comment by thatskymod** [comment]
No worries! As of version 1.4.8, both the Mod Manager and the game application have been combined into a single icon. This change addresses the issue where both icons were not displaying correctly on some devices.

Below is a brief explanation for future readers.

![Canvas-Merged-Issue-73](https://github.com/thatskymod/Sky-CotL-Scripts/assets/90126520/a83f2569-482f-421a-86f8-bb5170804435)
